pkg/kademlia/routing_helpers_test.go

```diff
@@ -349,24 +308,25 @@ func TestNodeIsWithinNearestK(t *testing.T) {
 			closest: true,
 		},
 		{testID: "C",
-			nodeID:  storj.NodeID{255, 255},
+			nodeID:  storj.NodeID{255, 255}, //closest to self
 			closest: true,
 		},
 		{testID: "D",
 			nodeID:  storj.NodeID{191, 255},
-			closest: true,
+			closest: false,
 		},
 		{testID: "E",
 			nodeID:  storj.NodeID{133, 255},
 			closest: false,
```

Comment:
if the bucketsize is 6 and we're adding 5 things, so are we ever not within K?

Reply:
Oh I wonder if that's new, I see it's 6 now, I thought it was 2 at one point. In that case, yeah this whole thing is wrong then.